src/datashare/azext_datashare/_help.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+
+from .generated._help import *  # noqa: F403
+try:
+    from .manual._help import *  # noqa: F403
+except ImportError:
+    pass
```

Comment:
What's wrong here? You disable linter?
If ImportError, can the command run as expected?

Reply:
I think this is to allow no manual `_help.py` if the generated `_help.py` is all good.